src/app/campaign/campaign.component.scss

```diff
@@ -29,3 +29,8 @@ mat-drawer {
   text-align: center;
   padding: 12px 0;
 }
+
+.mat-list-base .mat-list-item.deleted-flight {
+  text-decoration: line-through;
+  color: #f00;
```

Comment:
Not sure if you got this red color from the styleguide https://github.com/PRX/styleguide.prx.org/blob/master/projects/ngx-prx-styleguide/src/assets/styles/_variables.scss#L65, but FYI the emphasis colors have typically been reserved for hover/focus/active.